app-backend/batch/src/main/scala/com/azavea/rf/batch/export/spark/Export.scala

```diff
@@ -50,7 +50,7 @@ object Export extends SparkJob with Config with LazyLogging {
     projLocs: Map[UUID, List[(UUID, String)]],
     conf: HadoopConfiguration
   )(implicit sc: SparkContext): Unit = {
-    interpretRDD(ast, params.sources, ed.input.resolution, sceneLocs, projLocs) match {
+    interpretRDD(ast, params.sources, params.overrides, ed.input.resolution, sceneLocs, projLocs) match {
```

Comment:
Currently, it happens inside `LayerCache`: https://github.com/azavea/raster-foundry/blob/develop/app-backend/tile/src/main/scala/LayerCache.scala#L206-L214

For TMS, at least, I think we'll probably have to keep such replacements in this less convenient/readable/comprehensible location